.github/workflows/ci.yml

```
@@ -76,10 +76,7 @@ jobs:
   # Use cross for QEMU-based testing
   # cross needs to execute Docker, GitHub Action already has it installed
   cross:
-    # Still use 20.04 for this CI step as test `test_prctl::test_set_vma_anon_name`
-    # would fail on 22.04 and 24.04 (at least for now)
-    # https://github.com/nix-rust/nix/issues/2418
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

Generally pin a version is more stable. We can upgrade 24.04 when it's ready, but never accidently broken by a GitHub change.

---

Makes sense, I will do a cleanup of this